db_upgrade_scripts/mosip_audit/sql/1.1.5.5_to_1.2.0.1-B1_rollback.sql

```diff
@@ -8,4 +8,7 @@
 -- Modified Date        Modified By         Comments / Remarks
 -- -------------------------------------------------------------------------------------------------
 
-\c mosip_audit sysadmin
\ No newline at end of file
+\c mosip_audit sysadmin
+
+--This only need to execute if no rows added after upgradescripts for host_ip column---------------
+ALTER TABLE audit.app_audit_log ALTER COLUMN host_ip TYPE varchar(16) USING host_ip::varchar;
```

Comment:
If IPv6 is used, length might be high. Should we increase this to 50 rather ?

Reply:
updated